### DISCUSSION TITLE
How to properly set highlight group on git-blame?

### DISCUSSION THREAD

**Discussion by jakubkozlowicz** [LazyVim/LazyVim#1008] [Q&A]
Hi!

How to properly set highlight group colours for the git-blame plugin when using cursor line option?

I have following code for the plugin.

```lua
return {
  {
    "f-person/git-blame.nvim",
    event = "BufRead",
    config = function()
      vim.cmd("highlight default link gitblame SpecialComment")
      vim.g.gitblame_enabled = 0
      vim.g.gitblame_message_template = " <summary> • <date> • <author>"
    end,
  },
}
```

<img width="702" alt="image" src="https://github.com/LazyVim/LazyVim/assets/60330489/517b2429-2437-4087-bc31-65e7aa1bee0f">

As you can see the background of the blame message does not match the cursor line background. Is there any option to set matching styles?

I use the tokyonight moon colorscheme.

**Comment by folke** [comment]
This is not really the place to ask this. However, you first will have to figure out what hl group is used there. Check the blame.nvim docs.

Then check if you can change the group or if tokyonight already defines the group. Feel free to submit a PR with better hl groups for tokyonight
